doc/rst/source/gmt2kml.rst

```diff
@@ -256,15 +254,15 @@ Optional Arguments
 
```

Comment:
There's an extra bracket at the end of the -S description in line 226.

Reply:
Got it.